covid19_sfbayarea/data/san_mateo/time_series_cases.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+from typing import Any, Dict, List
+
+from .power_bi_querier import PowerBiQuerier
+
+class TimeSeriesCases(PowerBiQuerier):
+    def __init__(self) -> None:
+        self.source = 'c'
+        self.name = 'cases_by_day'
+        self.property = 'date_result'
+        super().__init__()
+
+    def _parse_data(self, response_json: Dict) -> List[Dict[str, Any]]:
+        data_pairs = super()._parse_data(response_json)
+        results = [ { 'date': self._timestamp_to_date(timestamp), 'cases': cases } for timestamp, cases in data_pairs ]
+        self._add_cumulative_data(results)
+        return results
+
+    def _timestamp_to_date(self, timestamp_in_milliseconds: int) -> str:
+        return datetime.utcfromtimestamp(timestamp_in_milliseconds / 1000).strftime('%Y-%m-%d')
+
+    def _add_cumulative_data(self, results: List[Dict[str, Any]]) -> None:
+        running_total = 0
+        for result in results:
+            running_total += result['cases']
+            result['cumul_cases'] = running_total
```

Comment:
Is there a separate, scrape-able data source for Total Cases by Day and New Cases by Day? The dashboard reports each separately: 
![image](https://user-images.githubusercontent.com/12928553/87175075-f59ae300-c28c-11ea-9d73-bc30d51d145c.png)

If they are listed separately, please scrape the cumulative Total Cases by Day as the county reports it. Today the cumulative numbers match, but sometimes they don't match up and we have decided to report just as the county reports it.

Reply:
Yes, they're listed separately, I will use the provided data rather than summing from the other data.

Interesting that they don't match up, I didn't expect this!